homeassistant/components/plex/media_browser.py

```diff
@@ -19,6 +29,15 @@
     "Recently Added": "recentlyAdded",
 }
 
+ITEM_TYPE_MEDIA_CLASS = {
+    "album": MEDIA_CLASS_ALBUM,
+    "artist": MEDIA_CLASS_ARTIST,
+    "playlist": MEDIA_CLASS_PLAYLIST,
+    "season": MEDIA_CLASS_SEASON,
+    "show": MEDIA_CLASS_TV_SHOW,
+    "video": MEDIA_CLASS_VIDEO,
+}
```

Comment:
Sorry, I meant to comment on all these types. Please check so we haven't missed anyone.